src/components/BoemlySteps/BoemlySteps.tsx

```diff
@@ -1,23 +1,36 @@
 import React from 'react';
-import { Flex, Text, useToken } from '@chakra-ui/react';
+import { Flex, Text, useToken, Stack } from '@chakra-ui/react';
 import { Check } from 'phosphor-react';
+import { ProgressDivider } from './styles';
 
 export interface BoemlyStepsProps {
   steps: {
     text: string;
     onClick: () => void;
   }[];
   currentStep: number;
+  horizontal?: boolean;
```

Comment:
I would maybe call it `orientation` and have two options. Like this:

```suggestion
  orientation?: 'vertical' | 'horizontal';
```

If you have a horizontal flag you always have to think: "Is the flag `horizontal` or `vertical`. If the flag is `orientation` it's always the same.